platforms/dir.targets

```diff
@@ -3,13 +3,13 @@
   <PropertyGroup>
     <OutDir>$(BaseOutputPath)ref/$(TargetGroup)/</OutDir>
     <GenerateProjectSpecificOutputFolder>false</GenerateProjectSpecificOutputFolder>
+    <SkipSigning>true</SkipSigning>
```

Comment:
Remove this property and after the import of dir.targets below define.

```
<!-- We don't need full signing for these so not writing the required signing file -->
<Target Name="WriteSigningRequired" />
```
Do the same thing for extensions targets.